Formula/disque.rb

```diff
@@ -0,0 +1,17 @@
+class Disque < Formula
+  desc "Disque, an in-memory, distributed job queue"
+  homepage "https://github.com/antirez/disque"
+  url "https://github.com/antirez/disque/archive/1.0-rc1.tar.gz"
+  version "1.0-rc1"
```

Comment:
We'll need to pass on this for now, then, sorry. Perhaps a candidate for https://github.com/homebrew/homebrew-devel-only?